From deaec2fda31ca2a3a30d146233ceebfbc6de647e Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Wed, 10 Apr 2013 16:31:11 +0200
Subject: [PATCH 07/11] BUG/MINOR: tcp: fix error reporting for TCP rules

tcp-request swapped two output words in the error message, making it meaningless.
---
 src/proto_tcp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/proto_tcp.c b/src/proto_tcp.c
index 781b24a..7c03798 100644
--- a/src/proto_tcp.c
+++ b/src/proto_tcp.c
@@ -1429,7 +1429,7 @@ static int tcp_parse_tcp_req(char **args, int section_type, struct proxy *curpx,
 		else
 			memprintf(err,
 			          "'%s' expects 'inspect-delay', 'connection', or 'content' in %s '%s' (got '%s')",
-			          args[0], args[1], proxy_type_str(curpx), curpx->id);
+			          args[0], proxy_type_str(curpx), curpx->id, args[1]);
 		goto error;
 	}
 
-- 
1.8.1.5

